dpe/fuzz/fuzz_targets/fuzz_target_1.rs

```diff
@@ -0,0 +1,12 @@
+// Licensed under the Apache-2.0 license
+
+#![no_main]
+
+use libfuzzer_sys::fuzz_target;
+
+mod runtime_dpe_dpe_fuzz_harness;
+use runtime_dpe_dpe_fuzz_harness::harness;
+
+fuzz_target!(|data: &[u8]| {
+    harness(data);
+});
```

Comment:
Do aft and fuzz need to be different crates? I would expect you could do something like:

```rust

#![no_main]

use afl;
use libfuzzer_sys;

mod runtime_dpe_dpe_fuzz_harness;
use runtime_dpe_dpe_fuzz_harness::harness;

libfuzzer_sys::fuzz_target!(|data: &[u8]| {
    harness(data);
});

afl::fuzz_target!(|data: &[u8]| {
    harness(data);
});
```

Reply:
Alternatively, if you think it's better to be able to run them separately, we can have a feature that can be enabled/disabled on the command-line.

Reply:
They have different function prototypes. fuzz is `no_main`, but afl actually expands the `fuzz!` macro inside `main()`. It might work though, I remember that the macro in libfuzzer-sys doesn't use `main` either. They're using what I think is called an intrinsic function?

I'll try it out.

Reply:
> Alternatively, if you think it's better to be able to run them separately, we can have a feature that can be enabled/disabled on the command-line.

Also, yes. As I remember commenting on one of the other PRs, when I was writing this, I hadn't extensively compared the two, and it was easy enough to support both, so I did. Which should be run, or whether we should run both - this is supported, by sharing a corpus directory, as I recall - is something to be discussed too.

Reply:
In that case we can probably have a feature. Could use `cfg_if`, since one needs no_main. I think that is preferable to defining two separate crates. Then when we run the test we can pass `--features=afl`

Reply:
I thought it would be possible to compile and link both together, since they have different entrypoints, but AFL packages its own LLVM runtime, causing linker errors. So, these are now both optional dependencies, one must be selected. This is resolved when I push updates next.

Reply:
Looks good, thanks!